worldedit-core/src/main/java/com/sk89q/worldedit/world/registry/BundledRegistries.java

```diff
@@ -65,6 +65,7 @@ static URL loadRegistry(String name) throws IOException {
     }
 
     private final BundledBlockRegistry blockRegistry = new BundledBlockRegistry();
+    @SuppressWarnings("removal")
```

Comment:
Shouldn't we deprecate this field / the related getters in this class?

Reply:
we can deprecate the field, but we cannot deprecate the getter currently because that propagates it to all classes that extend this, and would lead to non-deprecated methods getting marked as deprecated.

we currently can't remove this from the inheritance tree, so it's not really an option to deprecate this currently